pipeline/pipeline/dashboard_pdf_scrapper.py

```diff
@@ -113,18 +113,81 @@ def positive_breakdown_fix_dtypes(ward_positive_df):
     ward_positive_df["imputed"] = pd.to_numeric(ward_positive_df["imputed"])
     return ward_positive_df.copy()
 
+#! ==============================================
+#! ============ START TEMPORARY =======================
+#! ==============================================
+
+def _extract_wards_data_from_page(positive_cases_pdf_page) -> pd.DataFrame:
+    total_discharged_boundary = 610
+    discharged_deaths_boundary = 650
+    deaths_active_boundary = 700
+
+    # switching to our naming convention
+    # boxes = {
+    #     'ward': (240, 79.2, 350, 504),  # ward abbreviation
+    #     'total.confirmed': (350, 79.2, total_discharged_boundary, 504),  # cases
+    #     'total.recovered': (total_discharged_boundary, 79.2, discharged_deaths_boundary, 504),  # Discharged column
+    #     'total.deceased': (discharged_deaths_boundary, 79.2, deaths_active_boundary, 504),  # deaths column
+    #     'total.active': (deaths_active_boundary, 79.2, 800, 504),  # active column
+    # }
```

Comment:
just remove commented code... we can always drag it back form the github history